### DISCUSSION TITLE
Unable to open seb on windows 11

### DISCUSSION THREAD

**Discussion by LChiavegato** [SafeExamBrowser/seb-win-refactoring#322] [Q&A]
Hi i have students with windows 11 they have downloaded seb installer but they are unable
To open the installer. The message tells that it is unable to open the app on the pc and to find a compatible one.

**Comment by dbuechel** [comment]
We'd need the exact details of the error message, ideally with a print screen and the log file of the installer.

**Comment by LChiavegato** [comment]
Hi, it seems that with the last update o Win11 something with the registry going wrong. Only whit a complete deinstall and reinstall it works. Here the screen. I’ve also some problems with win 8.1, Fatal error Invalid name space
![win11](https://user-images.githubusercontent.com/97410165/148761789-7406c3ae-b4e1-4538-95c6-5149bf757d0b.jpg)
![win8 1](https://user-images.githubusercontent.com/97410165/148761809-8c503d0e-cd98-42c4-9f7e-f1a56cd962ea.jpg)
e

**Reply by dbuechel** [reply]
Okay, I'll check with my Windows 11 installation whether I can reproduce the behavior. Could you share the exact version information of your Windows 11 installation?

**Reply by dbuechel** [reply]
I can't reproduce the issue on my Windows 11 machine. We have planned a full release testing for Windows 11 so that we can officially support it, meanwhile we'd need more information about the underlying issue. Are there any entries in the Windows event logs? What is the exact Windows version (build) and edition?

Regarding the other error message, please refer to any of the already existing issues regarding this problem, e.g. #33.

**Comment by LChiavegato** [comment]
No can’t is a pc student


On 10 gennaio 2022 a 12:59:18, Damian Büchel ***@***.***)
scritto:

Okay, I'll check with my Windows 11 installation whether I can reproduce
the behavior. Could you share the exact version information of your Windows
11 installation?

—
Reply to this email directly, view it on GitHub
<https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/322#discussioncomment-1939707>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AXHFY5JYH7BWWDWWRUVUEDLUVLCZNANCNFSM5LTOEADA>
.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

You are receiving this because you authored the thread.Message ID:
<SafeExamBrowser/seb-win-refactoring/repo-discussions/322/comments/1939707@
github.com>

**Comment by LChiavegato** [comment]
Hi I don't have those information, as I can see, Seb was not removed
correctly. On Mac OS with seb 3.0 I see it’s possible to share the screen
during an exam, it’s possible to avoid this?


On 12 gennaio 2022 a 11:36:59, Damian Büchel ***@***.***)
scritto:

I can't reproduce the issue on my Windows 11 machine. We have planned a
full release testing for Windows 11 so that we can officially support it,
meanwhile we'd need more information about the underlying issue. Are there
any entries in the Windows event logs? What is the exact Windows version
(build) and edition?

Regarding the other error message, please refer to any of the already
existing issues regarding this problem, e.g. #33
<https://github.com/SafeExamBrowser/seb-win-refactoring/issues/33>.

—
Reply to this email directly, view it on GitHub
<https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/322#discussioncomment-1953863>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AXHFY5O62CJJHCMY6RKMX2TUVVKUXANCNFSM5LTOEADA>
.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

You are receiving this because you authored the thread.Message ID:
<SafeExamBrowser/seb-win-refactoring/repo-discussions/322/comments/1953863@
github.com>

**Reply by dbuechel** [reply]
Yes, there are some settings related to screen sharing, see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane

**Comment by LChiavegato** [comment]
Ok, but if I generate the profile via Moodle I don’t have those option, I
need to download the profile generate by Moodle and edit, and after
reupload? This is new on Mac on the 2.3.2 a shared screen goes black when
SEB is opening



On 13 gennaio 2022 a 16:17:37, Damian Büchel ***@***.***)
scritto:

Yes, there are some settings related to screen sharing, see
https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane

—
Reply to this email directly, view it on GitHub
<https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/322#discussioncomment-1963231>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AXHFY5PCVPQ3X57U5AY3T3LUV3UJDANCNFSM5LTOEADA>
.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

You are receiving this because you authored the thread.Message ID:
<SafeExamBrowser/seb-win-refactoring/repo-discussions/322/comments/1963231@
github.com>

**Reply by dbuechel** [reply]
Please open a new discussion for other, unrelated questions, and you'd especially need to ask macOS / iOS -related questions in the respective repository here on GitHub.

You can upload custom configurations to Moodle, so yes, that way it should work, but Moodle-related detail question would need to be asked on the Moodle forums. We unfortunately can't provide direct support here via GitHub, there is a last mile users need to go, reading the manual and attempting to figure things out by themselves.

**Comment by LChiavegato** [comment]
If I want to generate a model for Moodle may I use the desktop software?
Create a profile and use it as a model or i need to generate it only from
Moodle? I’ve checked the .sed file created by Moodle and the screen sharing
feature is not active but in google meet i can share the screen and see it,
so the exam is not secure as before.


On 13 gennaio 2022 a 16:22:22, luca chiavegato ***@***.***)
scritto:

Ok, but if I generate the profile via Moodle I don’t have those option, I
need to download the profile generate by Moodle and edit, and after
reupload? This is new on Mac on the 2.3.2 a shared screen goes black when
SEB is opening



On 13 gennaio 2022 a 16:17:37, Damian Büchel ***@***.***)
scritto:

Yes, there are some settings related to screen sharing, see
https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane

—
Reply to this email directly, view it on GitHub
<https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/322#discussioncomment-1963231>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AXHFY5PCVPQ3X57U5AY3T3LUV3UJDANCNFSM5LTOEADA>
.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

You are receiving this because you authored the thread.Message ID:
<SafeExamBrowser/seb-win-refactoring/repo-discussions/322/comments/1963231@
github.com>

**Reply by danschlet** [reply]
Moodle has the option "Require the use of Safe Exam Browser "Yes - Upload my own config", I guess you saw that. You can upload any SEB configuration file (generated with the Windows Configuration Tool or Mac Preferences, iOS Settings), as long as it's not encrypted (has to be saved with "Use config/settings file for .. starting an exam" and don't enter a settings password).

You have to add Google Meet to [Applications / Prohibited Processes](https://safeexambrowser.org/macosx/mac_usermanual_en.html#ProhibitedProcessesSection) or use the AAC kiosk mode ([Security / Use Assessment Mode (AAC) if available](https://safeexambrowser.org/macosx/mac_usermanual_en.html#SecurityPane)).

We will add Google Meet as a default prohibited process to the next SEB for macOS release. The AAC kiosk mode should block all screen sharing and recording tools, but it's only available on Macs running macOS Catalina 10.15.4 and doesn't work on some older macOS 11 Big Sur versions, as it had a bug there.
